src/Pages/HomePage.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Header } from '../Components/Header/Header.js';
+import { Main } from '../Components/Main/Main.js';
+import { Footer } from '../Components/Footer/Footer.js';
+
```

Comment:
make a single file and use that for import

Reply:
After finishing all my components I'll do